cypress/integration/001-start.js

```diff
@@ -0,0 +1,7 @@
+describe('Start page', function () {
+  it('main button should navigate to gdpr', function () {
+    cy.visit('/start');
+    cy.get('#main-content form button.govuk-button').click();
```

Comment:
Could you make all these tests work against a `.naturescot-forward-button` like we did in SFO?

https://github.com/Scottish-Natural-Heritage/standard-forestry-operations/blob/develop/cypress/integration/001-start.js#L4

Reply:
Resolved in 211c46070fa5c89d92c60d675396d54a5c4127dd.